internal/operands/common-templates/constants.go

```diff
@@ -11,6 +11,6 @@ const (
 	TemplateWorkloadLabelPrefix  = "workload.template.kubevirt.io/"
 	TemplateDeprecatedAnnotation = "template.kubevirt.io/deprecated"
 
-	CdiApiGroup = "cdi.kubevirt.io"
+	CdiApiGroup   = "cdi.kubevirt.io"
```

Comment:
Let's drop this change, unrelated to pr

Reply:
+1
I think gofmt should drop this anyways, don't we have a generation step for formating like in Kubevirt?

Reply:
Once this is fixed and CI is green, lgtm from me.

Reply:
> +1
> I think gofmt should drop this anyways, don't we have a generation step for formating like in Kubevirt?

We do: `make fmt` and `make vet`, they are also run on any other build target.

Reply:
This was done by either `make fmt` I guess, and it keeps coming back everytime, so I think it should be fixed

Reply:
Looked at this locally, I see what it is doing now, lining up the `=`. 👍